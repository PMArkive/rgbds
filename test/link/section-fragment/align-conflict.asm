IF !DEF(SECOND)
	def ATTRS equs ",ALIGN[2]"
ELSE
	def ATTRS equs "[$1337]"
ENDC

SECTION FRAGMENT "conflicting alignment", ROM0 {ATTRS}
	db

	PURGE ATTRS
