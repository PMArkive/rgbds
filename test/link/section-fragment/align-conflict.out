FATAL: Section "conflicting alignment" is defined with 4-byte alignment (offset 0), but also with address $1337
    at section-fragment/align-conflict.asm(7)
    and also:
    at section-fragment/align-conflict.asm(7)
Linking aborted with 1 error
---
error: Section already declared as aligned to 4 bytes (offset 0)
    at <stdin>(18)
FATAL: Cannot create section "conflicting alignment" (1 error)
    at <stdin>(18)
